rplugin/node/vim-package-info/parser.js

```diff
@@ -75,7 +77,13 @@ function getVersion(data, depSelector, dep, line) {
     else return line.trim();
   }
 
-  const verinfo = data[depSelector][dep];
+  // toml files will nest keys separated by a period, dependencies nested not at the top
+  // level, must be navigated to.
+  const pathItems = depSelector.split('.');
+  for (item of pathItems) {
```

Comment:
Initialize `item`. Just keeping it tidy.

```suggestion
  for (let item of pathItems) {
```